src/core.jl

```diff
@@ -0,0 +1,8 @@
+wrapping_type{CT<:Colorant}(::Type{CT}) = base_colorant_type(CT){FixedPointNumbers.floattype(eltype(CT))}
```

Comment:
I found a function[ `accum` ](https://github.com/JuliaImages/Images.jl/blob/master/src/algorithms.jl#L307)in `Images.jl` which serves the same purpose. I will use that instead.